src/newrelic.js

```diff
@@ -0,0 +1,200 @@
+/*
+ * Copyright 2019 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+/* eslint-disable no-console */
+
+const yargs = require('yargs');
+const fs = require('fs');
+const path = require('path');
+const request = require('request-promise-native');
+
+const frequency = 15;
+const status = 'ENABLED';
+const slaThreshold = 7;
+const locations = ['AWS_AP_NORTHEAST_1',
+  'AWS_AP_NORTHEAST_2',
+  'AWS_AP_SOUTH_1',
+  'AWS_AP_SOUTHEAST_1',
+  'AWS_AP_SOUTHEAST_2',
+  'AWS_CA_CENTRAL_1',
+  'AWS_EU_CENTRAL_1',
+  'AWS_EU_WEST_1',
+  'AWS_EU_WEST_2',
+  'AWS_EU_WEST_3',
+  'AWS_SA_EAST_1',
+  'AWS_US_EAST_1',
+  'AWS_US_EAST_2',
+  'AWS_US_WEST_1',
+  'AWS_US_WEST_2',
+  'LINODE_US_CENTRAL_1',
+  'LINODE_US_EAST_1',
+  'LINODE_US_WEST_1'];
+
+let packageName;
+
+try {
+  packageName = JSON.parse(fs.readFileSync('package.json')).name;
+} catch (e) {
+  packageName = undefined;
+}
+
+
+async function getMonitors(auth, monitorname, monitorid) {
+  try {
+    let more = true;
+    const loadedmonitors = [];
+    while (more) {
+      // eslint-disable-next-line no-await-in-loop
+      const response = await request.get(`https://synthetics.newrelic.com/synthetics/api/v3/monitors?limit=100&offset=${loadedmonitors.length}`, {
+        headers: {
+          'X-Api-Key': auth,
+        },
+        json: true,
+      });
+      if (response.count < 10) {
+        more = false;
+      }
+      loadedmonitors.push(...response.monitors);
+    }
+
+    const monitors = loadedmonitors.map(({ id, name }) => ({ id, name }));
+    if (monitorid) {
+      return monitors.filter((monitor) => monitor.id === monitorid);
+    }
+    if (monitorname) {
+      return monitors.filter((monitor) => monitor.name === monitorname);
+    } else {
+      return [];
+    }
+  } catch (e) {
+    console.error('Unable to retrieve monitors');
```

Comment:
Please check the log changes from @koraa's other branch.

Reply:
I'll wait for her branch to be merged 😄 

Although in the case of this type of tooling, i would argue that you expect to see the output in the `console`, no?